### PR TITLE
Solving #482

### PR DESCRIPTION
**Pull request by sim51**
See  #482

### CODE REVIEW

File: src/gexf/common/parser.js [hunk 1]
```diff
@@ -143,6 +143,8 @@ function collectAttributes(model, defaults, element) {
     valueElement = valueElements[i];
     id = valueElement.getAttribute('id') || valueElement.getAttribute('for');
 
+    if(!model[id]) throw new Error('Found attribut \'' + id + '\' which is not defined');
```
Comment (Yomguithereal) [inline src/gexf/common/parser.js:146]: The error message should be `graphology-gexf/parser: Found undeclared attribute "blabla".`

File: src/gexf/test/resources/index.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ var FILES = [
   'liststring',
   'minimal',
   'missing_nodes',
+  'missing_attribut_def',
```
Comment (Yomguithereal) [inline src/gexf/test/resources/index.js:21]: `undeclared_attribute`

File: src/gexf/test/browser/parser.js [hunk 3]
```diff
@@ -29,5 +29,20 @@ describe('Parser', function () {
     }, /inexisting/);
   });
 
+  it('should throw if an edge points to an inexisting node.', function () {
```
Comment (Yomguithereal) [inline src/gexf/test/browser/parser.js:32]: You duplicated this test from above.

**Review by Yomguithereal — CHANGES_REQUESTED** [review]

File: src/gexf/test/browser/parser.js (hunk 3 above)
Comment (sim51) [reply src/gexf/test/browser/parser.js]: oups, sorry

**Comment by Yomguithereal** [comment]
Thanks @sim51, I will merge soon and add some setting allowing undeclared attribute to be parsed even if no declaration can be found, in which case they will stay as strings.

**Review by Yomguithereal — APPROVED** [review]

**Pull request merged**